src/encoded/tests/test_clear_db_es_contents.py

```diff
@@ -22,56 +56,254 @@ def test_clear_db_tables(app, testapp):
     testapp.get(post_res.location, status=404)
 
 
-def test_run_clear_db_envs(app, testapp):
+_FOURFRONT_PRODUCTION_ENVS = ['fourfront-production-blue', 'fourfront-production-green', 'data', 'staging']
+# Really we only care about the first of these names, but the rest are names that were at one time
+# planned to be stg or prd names for cgap, so we'll use them to tell that run_clear_db_es is properly
+# skipping any such names. -kmp 4-Jun-2022
+_CGAP_PRODUCTION_ENVS = ['fourfront-cgap', 'fourfront-cgap-green', 'cgap-green', 'fourfront-cgap-blue', 'cgap-blue']
 
-    orig_env = app.registry.settings.get('env.name')
+_PRODUCTION_ENVS = _FOURFRONT_PRODUCTION_ENVS if is_running_app(APP_FOURFRONT) else _CGAP_PRODUCTION_ENVS
```

Comment:
Probably don't need to do this sort of thing and can just assign the FF names directly.

Reply:
I want to share the script between both repos.

Reply:
I have a plan in the back of my mind to pull these things out to a common repo that is actually shared.